### ISSUE TITLE
Issue sending multiple images/videos

### ISSUE THREAD

**Issue by pairofcrocs** [Iridium-IO/SiriShortcuts#6]
First let me start by saying, I really am finding a lot of use out of this shortcut. great work!

I'm noticing an issue however when it comes to sending multiple images/videos at the same time.

I try selecting multiple photos from the Photos app and it just hangs.

Just wondering if this is a limitation of the shortcut?

Thanks!

**Comment by Iridium-IO** [comment]
Yep it's a limitation, just one at a time for now unfortunately. I am working on multiple at once though :)
